python/graphstorm/model/gnn.py

```diff
@@ -622,7 +623,7 @@ def loss_func(self):
         return self._loss_fn
 
 def do_full_graph_inference(model, data, batch_size=1024, fanout=None, edge_mask=None,
-                            task_tracker=None):
+                            task_tracker=None, minibatch=False):
```

Comment:
i think it's better to not invoke mini-batch inference code inside full-graph inference.